app/src/main/java/com/github/dedis/student20_pop/PoPApplication.java

```diff
@@ -120,29 +117,22 @@ public Person getPerson() {
     /**
      * Returns the current LAO.
      */
-    public Lao getCurrentLao() {
-        return  currentLao;
+    public Optional<Lao> getCurrentLao() {
+        return Optional.ofNullable(currentLao);
     }
 
     /**
-     * Returns list of LAOs corresponding to the user
+     * @return the current LAO, unsafe
      */
-    public List<Lao> getLaos() {
-        return new ArrayList<>(laoEventsMap.keySet());
+    public Lao getCurrentLaoUnsafe() {
+        return currentLao;
```

Comment:
Is this one kept only for the tests? if so, maybe add a comment

Reply:
No, it is used whenever we are sure the current Lao is not null. Like in the Organizer and Attendee UI

Reply:
right, I saw that afterwards